CITATION.cff

```diff
@@ -0,0 +1,27 @@
+cff-version: 1.2.0
+message: "If you use Volcano in your work, please cite it using the information below. This helps us demonstrate the project's impact."
+title: "Volcano: A Cloud Native Batch System"
+authors:
+  - given-names: "Klaus"
+    family-names: "Ma"
+  - given-names: "Kevin"
+    family-names: "Wang"
+  - given-names: "Zenlin"
```

Comment:
Where does this come from?

Reply:
you are right- i made a mistake. I'll only keep the first two persons in maintainer.md file if its fine with you.